RecoTracker/TkSeedGenerator/plugins/SeedProducerFromSoA.cc

```diff
@@ -95,21 +97,20 @@ void SeedProducerFromSoAT<TrackerTraits>::produce(edm::StreamID streamID,
   // std::cout << "beamspot " << bsh.x0() << ' ' << bsh.y0() << ' ' << bsh.z0() << std::endl;
   GlobalPoint bs(bsh.x0(), bsh.y0(), bsh.z0());
 
-  const auto& tsoa = *(iEvent.get(tokenTrack_));
+  auto& tsoa = iEvent.get(tokenTrack_);
```

Comment:
I'd keep the explicit `const`
```suggestion
  auto const& tsoa = iEvent.get(tokenTrack_);
```